### PR TITLE
Add AI widget

### PR DESCRIPTION
**Pull request by JakeSCahill**
Adds the [Kapa AI widget](https://docs.kapa.ai/installation-widget#optional-configuration-parameters-) to our docs.

Preview: https://deploy-preview-179--docs-ui.netlify.app/

### CODE REVIEW

File: src/partials/algolia-script.hbs [hunk 1]
```diff
@@ -72,7 +72,7 @@ window.addEventListener('DOMContentLoaded', function() {
 
   autocomplete({
     container: '#autocomplete',
-    placeholder: 'Search for docs',
+    placeholder: 'Search',
```
Comment (Deflaimun) [inline src/partials/algolia-script.hbs:75]:
```suggestion
    placeholder: 'AI Search',
```

Comment (JakeSCahill) [reply src/partials/algolia-script.hbs:75]: This is the placeholder for Algolia search 😄

Comment (JakeSCahill) [reply src/partials/algolia-script.hbs:75]: It doesn't just search docs anymore.

Comment (Deflaimun) [reply src/partials/algolia-script.hbs:75]: Ahhh got it!

**Review by Deflaimun — APPROVED** [review]
Looks super good now, with the buttons in the same style

**Comment by JakeSCahill** [comment]
> Looks super good now, with the buttons in the same style

Thanks for the suggestion 🚀

**Review by Feediver1 — APPROVED** [review]
LGTM--thanks @JakeSCahill

**Pull request merged**